configs/yolo/train_yolov3_on_ipu.sh

```diff
@@ -0,0 +1,4 @@
+exp_name=yolov3_d53_320_273e_coco_ipu
```

Comment:
Recommend updating the README.md in this directory and pasting these commands in readme.
We can add a new section like `Run YOLOv3 on IPU`.
We do not need to add a new bash script here since the argument is simple.